### PR TITLE
Ported/Modified "Eris Computers" Console Sprites From VOREstation

### PR DESCRIPTION
**Pull request by AndrewMontagne**
Ported some console sprites from VOREstation, with extensive modifications - they can now merge to form console banks!

Many thanks to VOREstation for releasing their assets under Creative Commons!

blinkenlights2 gif

:cl: AndrewMontagne
tweak: New console sprites.
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
*swoon*

File: code/game/machinery/computer/_computer.dm [hunk 1]
```diff
@@ -45,17 +48,50 @@
...
 
+/obj/machinery/computer/proc/smooth_others()
+	for(dir in list(EAST,WEST))
+		var/obj/machinery/computer/other = locate(/obj/machinery/computer, get_step(src, dir))
+		if(!isnull(other))
+			spawn(1) other.smooth_self() //Delay it slightly for when this gets deleted
+
+/obj/machinery/computer/proc/smooth_self()
+	// For the fancy console bank sprites
+	if(copytext(icon_state,1,9) != "computer")
```
Comment (SamHPurp) [inline code/game/machinery/computer/_computer.dm]: This, uhh, seems a bad way to do this? 🤔

**Comment by imsxz** [comment]
be careful, you might send us down a slippery slope to porting VOREstation stuff

File: code/game/machinery/computer/_computer.dm [hunk 2]
```diff
@@ -45,17 +52,55 @@
 		icon_state = initial(icon_state)
 		update_icon()
 
+/obj/machinery/computer/proc/can_smooth()
+	// Determine whether we can smooth and form a console bank.
+	var/list/states = icon_states(icon)
+	var/teststate = icon_state + "-c"
+	return teststate in states
```
Comment (FlattestGuitar) [inline code/game/machinery/computer/_computer.dm]: If someone in ten years looks at this PR and goes "wow, he merged THAT shitcode?", they should know that I merged it *ironically*.

**Pull request merged**